gen/drivers/tables.go

```diff
@@ -22,6 +22,16 @@ func (tables Tables[C, I]) Get(name string) Table[C, I] {
 	panic(fmt.Sprintf("could not find table name: %s", name))
 }
 
+// Has checks if a table with the given name exists.
+func (tables Tables[C, I]) Has(name string) bool {
```

Comment:
If we're only using this in the tests, perhaps it can either


* Be converted to a stanalone function in the test file.
* Be moved to the tests file
* Be unexported

Every exported method is something that cannot be changed without a new version (breaking changes).   
Doesn't matter too much yet since Bob is not yet at v1.0, but it would matter more later.